combat-trainer.lic

```diff
@@ -2183,6 +2193,33 @@ class GameState
     end
   end
 
+  def tactics_analyze?
```

Comment:
The more I look at them, the more similar they appear to me (aside from the `return` cases at the top of the methods).

Why do they handle the `Analyze what?` case differently? They're both performing an `analyze`, just with or without an argument. Why does `perform_analyze?` not handle the `You must be closer...` case?